plugins/outputs/bigquery/bigquery.go

```diff
@@ -0,0 +1,227 @@
+package bigquery
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+const (
+	defaultOffSetKey   = "offset-key.json"
+	timeStampFieldName = "timestamp"
+)
```

Comment:
I think this isn't used.
```suggestion
const timeStampFieldName = "timestamp"
```